examples/java/src/main/java/org/apache/beam/examples/cookbook/JoinExamples.java

```diff
@@ -93,7 +93,7 @@
                   public void processElement(ProcessContext c) {
                     KV<String, CoGbkResult> e = c.element();
                     String countryCode = e.getKey();
-                    String countryName = "none";
+                    String countryName;
```

Comment:
Please combine this line with the next one

Reply:
Done!